packages/components/src/index.js

```diff
@@ -71,6 +72,7 @@ export { default as FormToggle } from './form-toggle';
 export { default as FormTokenField } from './form-token-field';
 export { default as __experimentalGradientPicker } from './gradient-picker';
 export { default as __experimentalCustomGradientPicker } from './custom-gradient-picker';
+export { default as __experimentalCustomGradientBar } from './custom-gradient-bar';
```

Comment:
Instead of exposing these two level components, would it make sense to move the DuoTone picker here and have a generic `DuotonePicker` component instead here?